source/_components/device_tracker.google_plus.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: page
+title: "Google Plus"
+description: "Instructions how to use Google Plus to track Android devices in Home Assistant."
+date: 2016-11-1 19:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Presence Detection
+---
+If you share your Android's location with Google, this component fetches those locations from Google's servers to your Home Assistant. This component does NOT directly communicate with your phone, thus using this component does not cause any additional battery drain.
+Suppose your Android phone is linked to your Google Account, say main@gmail.com.
+This component can be used in 2 modes: 
+- Self Mode: It visits "aboutme.google.com" while appearing (to Google) to be logged in as main@gmail.com. Using this mode requires putting your Google cookies into the home assistant config file. If anyone gets access to these cookies, they may be able to do anything with your google account, e.g. delete the account.
+- Safe Mode: In this safer mode, the component visits the Google Plus profile page of main@gmail.com, while appearing (to Google) to be logged in as a dummy Google account, say dummy@gmail.com. In this case, (only) the cookies of dummy@gmail.com are needed. Just for the initial setup, you need to install the Google Plus Android App on Android, and log into the app as main@gmail.com, and in its location settings, share the location with dummy@gmail.com. After that, you can uninstall the Google Plus app on Android.
+
+
+```yaml
+# Example configuration.yaml entry
+device_tracker 3:
+  platform: gplus
+  id: 'phone'
+  url: '...'
+  cookie_sid: '...'
+  cookie_hsid: '...'
+  cookie_ssid: '...'
+  data_freq: '...'
+  data_at: '...'
+```
+
```

Comment:
Overview of configuration variables are missing.

Example from the NUT UPS component.

```yaml
Configuration variables:

- **name** (*Optional*): Name prefix for defined sensors. Defaults to 'NUT UPS'.
- **host** (*Optional*): The host name or address of the device that is running NUT. Defaults to localhost.
- **port** (*Optional*): The port number. Defaults to 3493.
- **alias** (*Optional*): Name of the ups on the NUT server. Will default to the first ups name listed.
- **username** (*Optional*): Username to login to the NUT server. Default is none.
- **password** (*Optional*): Password to login to the NUT server. Default is none.
- **resources** array (*Required*): Contains all entries to display.
```